### DISCUSSION TITLE
Amapche.org link to api information is  broken

### DISCUSSION THREAD

**Discussion by Luud2000** [ampache/ampache#3237] [Q&A] [answered]
On the Ampache.org site the api link is broken, 404 error.

https://ampache.org/api/api/

Fixing that is greatly appreciated.

A suggestion  to where the documentation are also welcome.

**Comment by mitchray** [comment]
Is still this happening for you? The header API link is going to https://ampache.org/api/ for me

**Reply by lachlan-00** [reply]
not sure how that would happen. the links are all the same
![image](https://user-images.githubusercontent.com/1305249/166398073-9712ce9a-d307-4c5e-85ab-2267ad183d53.png)

**Comment by lachlan-00** [comment, marked as answer]
Documentation is built from the markdown here:
https://github.com/ampache/ampache.org-api/tree/master/content

which is built and then imported to the website and updated
https://ampache.org/api/

example docs are linked in the documentation but all generated and stored in the python3 library repo for each api version

There are example json and xml examples for each api version available
https://github.com/ampache/python3-ampache/tree/api5/docs
https://github.com/ampache/python3-ampache/tree/api4/docs
https://github.com/ampache/python3-ampache/tree/api3/docs

**Comment by Luud2000** [comment]
Hi it is working now.Thanks  Sent from ProtonMail for iOS Op EE, MMM d, jjjjj om HH:mm, Mitch Ray ***@***.***> schreef:  
Is still this happening for you? The header API link is going to https://ampache.org/api/ for me
